xcube/core/resampling/affine.py

```diff
@@ -22,10 +22,10 @@
 
 def affine_transform_dataset(
     source_ds: xr.Dataset,
-    *,
-    target_ds: Optional[xr.Dataset] = None,
+    /,
     source_gm: Optional[GridMapping] = None,
     target_gm: Optional[GridMapping] = None,
+    ref_ds: Optional[xr.Dataset] = None,
     var_configs: Optional[Mapping[Hashable, Mapping[str, Any]]] = None,
     encode_cf: bool = True,
     gm_name: Optional[str] = None,
```

Comment:
Default is None, but in the docstring below it says Defaults to "crs"

Reply:
What, "crs" ?

Reply:
Cannot find the place. Where does it say so?

Reply:
in Line 61, you need to expand it

Reply:
Ahh, that's ok. When `None` is passed, `"crs"` will be used, because it is optional. See tests.